### PR TITLE
Added Davy's Grey as a new Secondary Color

### PR DESCRIPTION
**Pull request by vkWeb**
#### Tasks accomplished: 
- [x] Added `#555555`(Davy's Grey) in the list of secondary colors. 

#### Description
This task was assigned to me by @QuincyLarson. Davy's grey is added to the list of secondary color because we're going to change the color of the timestamps of `beta.freecodecamp.org`. 

This is my first pull request on GitHub. @QuincyLarson I'll really appreciate your feedback. :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -306,6 +306,16 @@ <h4>White</h4>
               <p>#FFFFFF</p>
             </div>
           </div>
+
+          <!-- Adding a new secondary color #555555 - By @vkWeb -->
```
Comment (systimotic) [inline index.html]: Can you remove this comment too?

File: css/style.css [hunk 2]
```diff
@@ -206,3 +206,9 @@ nav ul li a:focus {
   background-color: #FFFFFF;
     border: 1px solid #D7E0E0;
 }
+
+/* Adding a new secondary color #555555 (Davy's Grey). It is used by map-block-time(timestamp) of beta.freecodecamp.org - By @vkWeb */
```
Comment (systimotic) [inline css/style.css]: Can you remove this comment? The credit will be saved in the git history. Not having this comment will save us the trouble of having to update it when things change (not new anymore, used in a different place, changed by somebody else).

File: css/style.css [hunk 3]
```diff
@@ -206,3 +206,9 @@ nav ul li a:focus {
...
+
+.davys-grey{
+background-color: #555555;
```
Comment (systimotic) [inline css/style.css]: Can you indent the `background-color` line like the declarations above?

**Review by systimotic — DISMISSED** [review]
Congratulations on your first PR @vkWeb! 🎉 

Travis CI is failing because it's not actually set up, so it uses its defaults which expect a Ruby project. This isn't your fault, you can ignore it. We should probably disable Travis CI here.

File: css/style.css (hunk 3 above)
Comment (vkWeb) [reply css/style.css]: Oh, my bad 😞 . I should have done that before.

File: index.html (hunk 1 above)
Comment (vkWeb) [reply index.html]: Sure. I'll do that.

File: css/style.css (hunk 2 above)
Comment (vkWeb) [reply css/style.css]: Sure. I wrote the comment just to clarify things. It wasn't meant for credit :)

**Comment by vkWeb** [comment]
@systimotic At first I thought that it was something wrong with my code but then looking at the details I realized it was not set up like you said. 

Thanks for the review :). I'll make the changes right now.

**Comment by virtual** [comment]
This might require another issue, but it'd be good to update the light font on the style guide to `#555555` as well, since it's not accessible :)

**Comment by vkWeb** [comment]
@virtual What issue are you talking about? Are you talking about the font-color of paragraphs? If Yes, then I'm with you as it fails AA compliant of [WCAG 2.0 guidelines for contrast accessibility](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html).

Here's a screenshot of the result with `#939393` when tested on [accessible-colors](http://accessible-colors.com/). **It fails**:
fails

Here's a screenshot of the result with `#555555` when tested on [accessible-colors](http://accessible-colors.com/). **It passes**:
passes

I won't recommend changing the background color. So, now we can do either of the following: 
- We can keep the font color same `#939393` by increasing the font weight to 700.
- We can change the font color to `#555555`. 

If @QuincyLarson agrees with us. I'm ready to raise a PR.

**Comment by QuincyLarson** [comment]
@vkWeb These changes look good! Thank you for your contribution!

**Pull request merged**